lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -187,75 +213,42 @@ define((require, exports, module) ->
 
     # For unknown reasons  we can"t use constants in block below, because background_tokenizer
```

Comment:
is tis comment still actual? Any sense to use constants as keys below (and probably booleans as values because current values seem to be the same as keys)